src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTestForFailedStatus.java

```diff
@@ -357,6 +357,37 @@ public void should_record_validation_failure_when_zip_duplicate_payment_dcns() t
         errorWasSent(SAMPLE_ZIP_FILE_NAME, ErrorCode.ERR_METAFILE_INVALID);
     }
 
+    @Test
+    public void should_record_validation_failure_when_zip_does_not_contain_required_document_type() throws Exception {
+        // given
+        uploadToBlobStorage(SAMPLE_ZIP_FILE_NAME, zipDir("zipcontents/missing_document")); // no "form" type document
```

Comment:
Please, rename missing_document to a more descriptive

Reply:
renamed the test